CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Features
 - Added two more submodels (options) for the SEI: Lars von Kolzenberg (2020) model and Tunneling Limit model ([#4394](https://github.com/pybamm-team/PyBaMM/pull/4394))
+- Enhanced the `search` method to accept multiple search terms in the form of a string or a list. ([#4650](https://github.com/pybamm-team/PyBaMM/pull/4650))
```

Comment:
Could you please move this to the top and add a line after "## Features" (we sort PRs in descending order)?